cmd/queue/main.go

```diff
@@ -114,6 +115,10 @@ func initEnv() {
 func reportStats(statChan chan *autoscaler.Stat) {
 	for {
 		s := <-statChan
+		if !userContainerDeployed {
```

Comment:
You should be able to use `healthState` here I think. It will only report success if the user containers was successfully probed as well.